packages/flutter/test/widgets/image_test.dart

```diff
@@ -324,7 +324,7 @@ class TestImageProvider extends ImageProvider<TestImageProvider> {
   }
 
   @override
-  String toString() => '$runtimeType($hashCode)';
+  String toString() => '$runtimeType#$hashCode';
```

Comment:
technically this should be `'$runtimeType#$hashCode'()` since this hierarchy sometimes has data to show